cli/build.rs

```diff
@@ -40,9 +40,7 @@ fn main() -> std::io::Result<()> {
 
     println!(
         "UPDATE_SCHEMA = {}",
-        env::var("UPDATE_SCHEMA")
-            .unwrap_or_else(|_| String::from("null"))
-            .as_str()
+        &env::var("UPDATE_SCHEMA").unwrap_or_else(|_| String::from("null"))
```

Comment:
I'm still confused on `&String` vs. `&str` -- I know the latter is a slice, and I understand the difference between the two, but I'm not yet sure what I should be using one or the other when _I_ declare the type.

Reply:
in many cases `&String` can act as `&str` because it impls `Deref<Target = str>` - in these situations, the `&` feels less obtrusive to what is happening in the logic